packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -303,29 +395,66 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
             setIsOpen(false);
           }
         }}
+        setFocus={!multiple}
       >
         <div
           ref={selectRef}
           className={cx('iui-select-button', {
-            'iui-placeholder': !selectedItem && !!placeholder,
+            'iui-placeholder':
+              (!selectedItems || selectedItems.length === 0) && !!placeholder,
             'iui-disabled': disabled,
             [`iui-${size}`]: !!size,
           })}
           onClick={() => !disabled && toggle()}
           onKeyDown={(e) => !disabled && onKeyDown(e, toggle)}
           tabIndex={!disabled ? 0 : undefined}
         >
-          {!selectedItem && <span className='iui-content'>{placeholder}</span>}
-          {selectedItem &&
-            selectedItemRenderer &&
-            selectedItemRenderer(selectedItem)}
-          {selectedItem && !selectedItemRenderer && (
+          {(!selectedItems || selectedItems.length === 0) && (
+            <span className='iui-content'>{placeholder}</span>
+          )}
+          {isMultipleEnabled(selectedItems, multiple) ? (
+            <>
+              {/* Either render custom multiple selected items provided by user */}
+              {isMultipleSelectedItemRenderer(selectedItemRenderer, multiple) &&
+                selectedItems &&
+                selectedItemRenderer(selectedItems)}
```

Comment:
Maybe we should pass `visibleCount` so users could handle that on their end if they wish.

Reply:
Just then need to figure where to put container ref, cause its needed for overflow to work.
Currently it goes on tag container, which is not present in custom renderer case

Reply:
Hm, if users are passing the container, maybe then we should leave overflow handling to them.

Reply:
they could pass, or they could not (like in story example).
So I dont know...